independent-publisher-connectors/Atlassian Jira/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1002 @@
+                },
+                "displayName": "Mia Krystof",
+                "active": true,
+                "timeZone": "Australia/Sydney",
+                "groups": {
+                  "size": 3,
+                  "items": []
+                },
+                "applicationRoles": {
+                  "size": 1,
+                  "items": []
+                }
+              }
+            },
+            "headers": {}
+          },
+          "401": {
+            "description": "Returned if the authentication credentials are incorrect or missing.",
+            "headers": {}
+          },
+          "403": {
+            "description": "Returned if the calling user does not have the *Browse users and groups* global permission.",
+            "headers": {}
+          },
+          "404": {
+            "description": "Returned if:\n\n *  `accountId` is missing.\n *  the user is not found.",
+            "headers": {}
+          }
+        }
+      }
+    },
+    "/rest/webhooks/1.0/webhook": {
+      "x-ms-notification-content": {
+        "description": "Project Create Response",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "self": {
+              "type": "string",
+              "description": "self"
+            },
+            "id": {
+              "type": "string",
+              "description": "id"
+            },
+            "key": {
+              "type": "string",
+              "description": "key"
+            },
+            "name": {
+              "type": "string",
+              "description": "name"
+            },
+            "avatarUrls": {
+              "type": "object",
+              "properties": {
+                "48x48": {
+                  "type": "string",
+                  "description": "48x48"
+                },
+                "24x24": {
+                  "type": "string",
+                  "description": "24x24"
+                },
+                "16x16": {
+                  "type": "string",
+                  "description": "16x16"
+                },
+                "32x32": {
+                  "type": "string",
+                  "description": "32x32"
+                }
+              },
+              "description": "avatarUrls"
+            },
+            "projectCategory": {
+              "type": "object",
+              "properties": {
+                "self": {
+                  "type": "string",
+                  "description": "self"
+                },
+                "id": {
+                  "type": "string",
+                  "description": "id"
+                },
+                "name": {
+                  "type": "string",
+                  "description": "name"
+                },
+                "description": {
+                  "type": "string",
+                  "description": "description"
+                }
+              },
+              "description": "projectCategory"
```

Comment:
@Flow-Joe Can you please provide description values as description values should be free from grammatical and spelling mistakes.

Reply:
@Flow-Joe Applies to all the similar properties